flepimop/gempyor_pkg/src/gempyor/dev/dev_seir.py

```diff
@@ -35,7 +35,7 @@
     out_prefix=prefix,
 )
 
-seeding_data = modinf.seeding.get_from_config(sim_id=100, modinf=modinf)
+seeding_data = modinf.get_seeding_data(100)
```

Comment:
echoing @pearsonca, I really think there is some interest in keeping everything as named arguments as for now there is not much a common scheme across modules

Reply:
https://github.com/HopkinsIDD/flepiMoP/pull/422#discussion_r1907518648